scipy/stats/_multivariate.py

```diff
@@ -3850,3 +3851,431 @@ def rvs(self, dim, size=1, random_state=None):
 
 
 unitary_group = unitary_group_gen()
+
+_mhg_doc_default_callparams = """\
+m : array_like
+    The number of different types of objects
+    in the population.
```

Comment:
```suggestion
    The number of each type of object in the population.
```

Right? As written, it might be confused as "the number of (distinct types)"
Could add something like:
```
    That is, `m[i]` is the number of objects of type `i`. 
```
for additional clarification.

Reply:
yes, you are right! I have added the extra clarification

Reply:
Please also correct the first part. "The number of each type of object in the population." is unambiguous; I think "The number of different types of objects in the population." will usually be interpreted differently than you mean.

Reply:
sorry, missed it before. done now